_posts/0000-01-02-Hazkiss.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"Some people think they can outsmart me. Maybe... maybe. I have yet to meet one that can outsmart bullet."